src/app/shared/object-geospatial-map/object-geospatial-map.component.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  isPlatformBrowser,
+  NgIf,
+} from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  Input,
+  PLATFORM_ID,
+  ViewEncapsulation,
+} from '@angular/core';
+import { Item } from 'src/app/core/shared/item.model';
+import { getItemPageRoute } from 'src/app/item-page/item-page-routing-paths';
+
+import { environment } from '../../../environments/environment';
+import { PaginatedList } from '../../core/data/paginated-list.model';
+import { RemoteData } from '../../core/data/remote-data';
+import { ViewMode } from '../../core/shared/view-mode.model';
+import { fadeIn } from '../animations/fade';
+import { hasValue } from '../empty.util';
+import { GeospatialMapComponent } from '../geospatial-map/geospatial-map.component';
+import { GeospatialMapDetail } from '../geospatial-map/models/geospatial-map-detail.model';
+import { ItemSearchResult } from '../object-collection/shared/item-search-result.model';
+import { ListableObject } from '../object-collection/shared/listable-object.model';
+import { parseGeoJsonFromMetadataValue } from '../utils/geospatial.functions';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.Default,
+  encapsulation: ViewEncapsulation.Emulated,
+  selector: 'ds-object-geospatial-map',
+  styleUrls: ['./object-geospatial-map.component.scss'],
+  templateUrl: './object-geospatial-map.component.html',
+  animations: [fadeIn],
+  standalone: true,
+  imports: [ GeospatialMapComponent, NgIf ],
+})
+
+export class ObjectGeospatialMapComponent {
```

Comment:
Please add a description to this component in a comment or TypeDocs.

Reply:
done